src/engine.c

```diff
@@ -22,6 +22,7 @@
 #include <config.h>
 #endif
 
+#include <stdio.h>
```

Comment:
Minor clean up thing. If this include is for ```atoi()```, the header for ```atoi()``` is ```<stdlib.h>```.  (But I built this commit with removing this line.  Maybe one of the other headers already has it. 🤔)

Reply:
This is for `sscanf ()`, without this header, gcc 9.1.1 will print out warnings.

Reply:
Sorry I missed the warning. You are right.

It works as intended. 👍

Reply:
No problem. Thanks for the review!